src/rendering/renderable.cpp

```diff
@@ -71,6 +79,20 @@ namespace {
 
         // [[codegen::verbatim(RenderableTypeInfo.description)]]
         std::optional<std::string> type;
+
+        // Fragile! Keep in sync with documentation
+        enum class [[codegen::map(openspace::Renderable::RenderBin)]] RenderBinMode {
+            Background,
+            Opaque,
+            PreDeferredTransparent,
+            PostDeferredTransparent,
+            Overlay
+        };
+
+        // [[codegen::verbatim(RenderableRenderBinModeInfo.description)]]
+        std::optional<RenderBinMode> renderBinMode [[codegen::key("RenderBinMode")]];
```

Comment:
This `key` definition is superfluous since the first letter of the variable name is capitalized automatically.